sql/core/src/main/scala/org/apache/spark/sql/execution/python/ArrowPythonRunner.scala

```diff
@@ -58,18 +58,18 @@ class ArrowPythonRunner(
     new WriterThread(env, worker, inputIterator, partitionIndex, context) {
 
       protected override def writeCommand(dataOut: DataOutputStream): Unit = {
-        PythonUDFRunner.writeUDFs(dataOut, funcs, argOffsets)
-        if (respectTimeZone) {
-          PythonRDD.writeUTF(timeZoneId, dataOut)
-        } else {
-          dataOut.writeInt(SpecialLengths.NULL)
+        dataOut.writeInt(conf.size)
```

Comment:
maybe put this in a  `writeConf` method to be more explicit?

Reply:
I think it's fine, but I will add some comments

Reply:
Ok, SGTM.